### PR TITLE
chore: Translation Fixes, Lint Error Corrections, and Additional Translations

### PR DESCRIPTION
**Pull request by berry-13**
### Fixed translation & small lint error, added some translation and removed console log for PaLM

- [x] Bug fix (non-breaking change which fixes an issue)
### Succesfull test:

npm run test:client
npm run e2e
npm run lint (no new error)

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Berry-13** [comment]
@danny-avila  Check if everything is alright, I have removed and modified some things. Let me know what you think and what doesn't work.

File: client/src/components/Input/EndpointMenu/EndpointItem.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Settings } from 'lucide-react';
 import { DropdownMenuRadioItem } from '~/components';
 import { getIcon } from '~/components/Endpoints';
 import { SetTokenDialog } from '../SetTokenDialog';
+import { localize } from '~/localization/Translation';
```
Comment (danny-avila) [inline client/src/components/Input/EndpointMenu/EndpointItem.tsx]: use the useLocalize hook instead

File: client/src/components/Input/EndpointMenu/EndpointMenu.jsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ import {
 } from '~/components/ui/';
 import DialogTemplate from '~/components/ui/DialogTemplate';
 import { cn, cleanupPreset, getDefaultConversation } from '~/utils';
+import { localize } from '~/localization/Translation';
```
Comment (danny-avila) [inline client/src/components/Input/EndpointMenu/EndpointMenu.jsx]: use the useLocalize hook instead

File: client/src/components/Input/Footer.tsx [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import { useGetStartupConfig } from 'librechat-data-provider';
+import { localize } from '~/localization/Translation';
```
Comment (danny-avila) [inline client/src/components/Input/Footer.tsx]: use the useLocalize hook instead

File: client/src/components/Plugins/Store/PluginAuthForm.tsx [hunk 4]
```diff
@@ -23,10 +23,10 @@ function PluginAuthForm({ plugin, onSubmit }: TPluginAuthFormProps) {
           className="col-span-1 flex w-full flex-col items-start justify-start gap-2"
           method="POST"
           onSubmit={handleSubmit((auth) =>
-            onSubmit({ pluginKey: plugin!.pluginKey, action: 'install', auth }),
+            onSubmit({ pluginKey: plugin?.pluginKey, action: 'install', auth }),
```
Comment (danny-avila) [inline client/src/components/Plugins/Store/PluginAuthForm.tsx:26]: has this been tested with a plugin with/without authentication needed?

File: client/src/components/Plugins/Store/PluginStoreDialog.tsx [hunk 5]
```diff
@@ -85,7 +85,7 @@ function PluginStoreDialog({ isOpen, setIsOpen }: TPluginStoreDialogProps) {
     setSelectedPlugin(getAvailablePluginFromKey);
 
     if (
-      getAvailablePluginFromKey!.authConfig.length > 0 &&
+      getAvailablePluginFromKey?.authConfig.length > 0 &&
```
Comment (danny-avila) [inline client/src/components/Plugins/Store/PluginStoreDialog.tsx:88]: has this been tested with a plugin with/without authentication needed?

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: client/src/components/Plugins/Store/PluginAuthForm.tsx (hunk 4 above)
Comment (berry-13) [reply client/src/components/Plugins/Store/PluginAuthForm.tsx:26]: I have tried installing and uninstalling the plugin with and without authentication, and it works. if you want, I can go back to the previous code.

**Review by danny-avila — APPROVED** [review]

**Pull request merged**